### ISSUE TITLE
Fix accessibility issues on Header/Nav

### ISSUE THREAD

**Issue by craigyu** [bcgov/nr-spar#128]
This task aims to search for any a11y issue on the Header/Nav and fix it. This task should be updated with each issue we find, so we can keep track of it for future pages that may contain the same issues.
 * Skip to main not working
 * Not announcing on screen reader when open panels
 * Right side menus not announcing on screen reader when tab is closed
 * Nav items not accessible via keyboard
